libs/clients/syslumenn/src/lib/syslumennClient.service.ts

```diff
@@ -356,16 +354,12 @@ export class SyslumennService {
     const res = await api.vedbokavottordRegluverkiPost({
       skilabod: {
         audkenni: id,
-        fastanumer:
-          propertyNumber[0] == 'F'
-            ? propertyNumber.substring(1, propertyNumber.length)
-            : propertyNumber,
+        fastanumer: cleanPropertyNumber(propertyNumber),
```

Comment:
Nice job, it's good to have the "clearnPropertyNumber" functionality in one place :)